frontend/app/src/pages/BuySell/AddLiquidity.tsx

```diff
@@ -48,10 +48,10 @@ const AddLiquidityComponent: React.FC<WithTranslation> = ({ t }) => {
 
   const calcMaxToken = (slippage: number, cashDeposited: number) => {
     if (cfmmStorage) {
-      const { tokenPool, cashPool } = cfmmStorage;
+      const { tezPool, cashPool } = cfmmStorage;
```

Comment:
are you sure about the order?

Reply:
yes, this is okay. it is named destructuring. the order doesn't matter.